### ISSUE TITLE
Sagemath Support 

### ISSUE THREAD

**Issue by cervera-queiroz** [twibiral/obsidian-execute-code#127]
Hey guys,

Is there any possibility to include Sagemath as one of the supported languages?

Thanks

**Comment by chlohal** [comment]
Sure! 

From a quick look at their docs, it seems like Sage doesn't have any way to execute a file non-interactively. Is there any way to do this, or would most Sagemath users prefer for it to only be available in Notebook Mode?

**Comment by chlohal** [comment]
Hi! I just wanted to follow up on this. Sagemath seems really interesting, and it's definitely something that I would like to contribute support for. The only issue is the requirement for interactive code: if sage could run a script file, then it would be very easy; do you know if that's possible?

**Comment by pelegm** [comment]
Yes, it is possible. E.g., `test.sage` may contain `print(3)`, and then running `sage test.sage` prints `3`.
